src/state/image-dataset/csv-dataset/index.ts

```diff
@@ -83,25 +84,74 @@ type FeatureInfo =
           data: (number | null)[];
       };
 
+type FeatureData = (number | null)[] | (string | null)[];
+
+function isNullOrNumericString(value: string | null): boolean {
+    if (value === null) {
+        return true;
+    } else if (typeof value != "string") {
+        return false;
+    } else if (value.trim().toLowerCase() === "nan") {
+        return true;
```

Comment:
should this return true??

Reply:
Updated the comments here for clarity on behavior.